packages/client/katello-pull-transport-migrate/katello-pull-transport-migrate.spec

```diff
@@ -0,0 +1,43 @@
+Name:           katello-pull-transport-migrate
+Version:        1.0.0
+Release:        1%{?dist}
+Summary:        An RPM that migrates katello-agent users to the new pull transport
+BuildArch:      noarch
+
+License:        LGPLv2
+URL:            https://github.com/theforeman/foreman-packaging/tree/rpm/develop/packages/client/katello-pull-transport-migrate
+Source0:        https://github.com/theforeman/%{name}/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+
+Requires:       yggdrasil
+Requires:       foreman_ygg_worker
+
+%description
+An RPM that migrates katello-agent users to the new pull transport.
+
+Relies on the existing Red Hat Subscription Manager configuration to know which Katello server it should communicate with.
+The scriptlet will fail if the client is not registered to a Foreman server with the Katello plugin.
+This RPM obtains the relevant values from the existing RHSM configuration, writes the configuration to yggdrasil's config.toml,
+and starts the pull transport agent.
+
+%prep
+%setup -q -c
+
+%build
+
+%install
+rm -rf %{buildroot}
```

Comment:
ain't nobody needs that in the year 2022

Reply:
Sounds like a Conan O'Brien skit :smile: